packages/ui-extensions/src/surfaces/admin/components/patterns/examples/mediaCard.html

```diff
@@ -0,0 +1,23 @@
+<s-box border="base" borderRadius="base" overflow="hidden">
+  <s-clickable href="#" accessibilityLabel="4-pieces puzzle template">
+    <s-image
```

Comment:
This one looks weird on my machine. I wonder if we need to constrain it somehow?

<img width="1085" height="936" alt="image" src="https://github.com/user-attachments/assets/960190d2-b4a7-425d-864a-1374d83d02f0" />